spaces/S000178/properties/P000004.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000178
+property: P000004
+value: true
+refs:
+  - doi: 10.1016/0016-660X(72)90026-8
+    name: A topological view of P-spaces (A. Misra)
+---
+
+Shown in Example 3.1 of {{doi:10.1016/0016-660X(72)90026-8}}.
```

Comment:
"Shown in ...": nothing is shown in the paper, it is just claimed there (not that it's necessarily difficult).  So how about just saying "See Example ..."?

(same comment applies to the other properties)